charts/istio/istio-istiod/templates/priorityclass-istiod.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: {{ include "priorityclassversion" . }}
+kind: PriorityClass
+metadata:
+  name: {{ .Values.priorityClassName }}
+value: 1000000000
```

Comment:
What is the impact of having istiod down actually? I am trying to understand whether it needs such high priority value.

Reply:
Istiod is responsible for configuring the envoys. I would give it the same priority as the istio-ingress-gateway (controlplane and dataplane).

Reply:
But still the question from https://github.com/gardener/gardener/pull/5468#discussion_r811162200 is not answered. What is the impact of having istiod down? Can istio-ingressgateway run without having istiod running?

Reply:
The envoys are running with last configured config from istiod. When istiod is unavailable for a certain period of time the istio ingress gateway will still work with the last applied configuration.

Reply:
If new istio-ingress-gateway pods are scheduled and istiod is not running in order to configure them we still have a broken setup, therefore I would use here a different priority.